crud/storage_info.lua

```diff
@@ -0,0 +1,117 @@
+local checks = require('checks')
+local errors = require('errors')
+local fiber = require('fiber')
+local log = require('log')
+
+local const = require('crud.common.const')
+local utils = require('crud.common.utils')
+
+local StorageInfoError = errors.new_class('StorageInfoError')
+
+local storage_info = {}
+
+local STORAGE_INFO_FUNC_NAME = 'storage_info_on_storage'
+local CRUD_STORAGE_INFO_FUNC_NAME = utils.get_storage_call(STORAGE_INFO_FUNC_NAME)
+
+--- Storage status information.
+--
+-- @function storage_info_on_storage
+--
+-- @return a table with storage status.
+local function storage_info_on_storage()
+    return {status = "running"}
+end
+
+storage_info.storage_api = {[STORAGE_INFO_FUNC_NAME] = storage_info_on_storage}
+
+--- Polls replicas for storage state
+--
+-- @function call
+--
+-- @tparam ?number opts.timeout
+--  Function call timeout
+--
+-- @tparam ?string|table opts.vshard_router
+--  Cartridge vshard group name or vshard router instance.
+--
+-- @return a table of storage states by replica id.
+function storage_info.call(opts)
+    checks({
+        timeout = '?number',
+        vshard_router = '?string|table',
+    })
+
+    opts = opts or {}
+
+    local vshard_router, err = utils.get_vshard_router_instance(opts.vshard_router)
+    if err ~= nil then
+        return nil, StorageInfoError:new(err)
+    end
+
+    local replicasets, err = vshard_router:routeall()
+    if replicasets == nil then
+        return nil, StorageInfoError:new("Failed to get router replicasets: %s", err.err)
+    end
```

Comment:
O am I missing something?

https://www.tarantool.io/ru/doc/latest/reference/reference_rock/vshard/vshard_router/#lua-function.vshard.router.routeall
```suggestion
    local replicasets, err = vshard_router:routeall()
    if err ~= nil then
        return nil, StorageInfoError:new("Failed to get router replicasets: %s", err.err)
    end
```

Reply:
Yeah, it seems that we had missed it on the original PR review. Updated. This commit only moves existing code, so I had put improvements to a separate commit "storage: refactor replicasets process on info".